packages/app/playwright/commons/gas.ts

```diff
@@ -4,7 +4,7 @@ import type { Provider, BN } from 'fuels';
 // TODO: remove this function when SDK make transactions with correct gas configs
 export const getGasConfig = async (provider: Provider) => {
   const chain = await provider.getChain();
-  const nodeInfo = await provider.getNodeInfo();
+  const nodeInfo = await provider.fetchNode();
```

Comment:
why you need to `fetchNode`? now when you do `await Provider.create()` it will do this request and cache for you. you can syncly access it:

```suggestion
  const nodeInfo = provider.getNode();
```